templates/blog/article.html

```diff
@@ -97,15 +97,23 @@ <h5>
 {% endblock %}
 
 {% block scripts %}
-  <script src="{{ static_url('js/dist/public.js') }}"></script>
-  <script>
-    Raven.context(function () {
-      try {
-        snapcraft.public.seriesPosts('[data-js="blog-series"]', '#blog-series-item-template');
-      } catch(e) {
-        console.error(e);
-        Raven.captureException(e);
-      }
-    });
-  </script>
+  {% set hasScript = false %}
```

Comment:
Feels a bit too much code for the template. It would work better as a context variable calculated in the view (`has_series` or something).